src/benchmarks/micro/corefx/System.Runtime/Perf.Enum.cs

```diff
@@ -20,19 +22,534 @@ public enum Colors
+        [Arguments(typeof(SmallContiguousEnum), (int)SmallContiguousEnum.One)]
+        public string GetNameUnderlyingDefined(Type enumType, object value) => Enum.GetName(enumType, value);
+
+        [Benchmark]
+        [Arguments(typeof(BigContiguousEnum), 36)]
+        [Arguments(typeof(SmallContiguousEnum), 3)]
+        public string GetNameUnderlyingUndefined(Type enumType, object value) => Enum.GetName(enumType, value);
+        #endregion
+
+        #region ToObject
+        [Benchmark]
+        [Arguments(typeof(BigContiguousEnum), 7)]
+        public object ToObject(Type enumType, object value) => Enum.ToObject(enumType, value);
+
+        [Benchmark]
+        [Arguments(typeof(BigContiguousEnum), 7)]
+        public object ToObjectInt32(Type enumType, int value) => Enum.ToObject(enumType, value);
+        #endregion
+
+        #region Equals
+        [Benchmark]
+        [Arguments(BigContiguousEnum.G, BigContiguousEnum.G)]
+        public bool EqualsTrue(Enum value, object obj) => value.Equals(obj);
+
+        [Benchmark]
+        [Arguments(BigContiguousEnum.G, BigContiguousEnum.H)]
+        public bool EqualsFalse(Enum value, object obj) => value.Equals(obj);
+
+        [Benchmark]
+        [Arguments(BigContiguousEnum.G)]
+        public bool EqualsNullFalse(Enum value) => value.Equals(null);
+
+        [Benchmark]
+        [Arguments(BigContiguousEnum.G, (int)BigContiguousEnum.G)]
+        public bool EqualsUnderlyingFalse(Enum value, object obj) => value.Equals(obj);
+        #endregion
+
+        [Benchmark]
+        [Arguments(BigContiguousEnum.N)]
+        public int GetHashCode(Enum value) => value.GetHashCode();
+
+        #region ToString
+        [Benchmark]
+        [Arguments(BigContiguousEnum.C)]
+        [Arguments(SmallContiguousEnum.One)]
+        public string ToStringDefined(Enum value) => value.ToString();
+
+        [Benchmark]
+        //[Arguments((BigContiguousEnum)36)] // BenchmarkDotNet Issue #1020
+        //[Arguments((SmallContiguousEnum)3)]
+        //public string ToStringUndefined(Enum value) => value.ToString();
+        public string ToStringUndefined() => ((BigContiguousEnum)36).ToString();
+
+        [Benchmark]
+        //[Arguments(Colors.Red | Colors.Orange | Colors.Yellow | Colors.Green | Colors.Blue)] // BenchmarkDotNet Issue #1020
+        //public string ToStringFlags5(Enum value) => value.ToString();
+        public string ToStringFlags5() => (Colors.Red | Colors.Orange | Colors.Yellow | Colors.Green | Colors.Blue).ToString();
+
+        [Benchmark]
+        //[Arguments(Colors.Orange | Colors.Yellow | Colors.Green)] // BenchmarkDotNet Issue #1020
+        //public string ToStringFlags3(Enum value) => value.ToString();
+        public string ToStringFlags3() => (Colors.Orange | Colors.Yellow | Colors.Green).ToString();
+
+        [Benchmark]
+        [Arguments(Colors.Red)]
+        public string ToStringFlags1(Enum value) => value.ToString();
```

Comment:
please update to `0.11.4.1025` and merge `ToStringFlags5`, `ToStringFlags3` and `ToStringFlags1` into one benchmark:

```cs
[Benchmark]
[Arguments(Colors.Red | Colors.Orange | Colors.Yellow | Colors.Green | Colors.Blue)]
[Arguments(Colors.Orange | Colors.Yellow | Colors.Green)]
[Arguments(Colors.Red)]
public string ToStringFlags(Enum value) => value.ToString();
```

The benchmark ID contains all the argument values so there is no need to have a separate method for that